python/cli/prelude_cli/views/iam.py

```diff
@@ -14,27 +14,42 @@ def iam(ctx):
     ctx.obj = IAMController(account=ctx.obj)
 
 
-@iam.command("create-account")
+@iam.command("account")
 @click.pass_obj
 @pretty_print
-@click.confirmation_option(
-    prompt="Overwrite local account credentials for selected profile?"
-)
-def register_account(controller):
-    """Register a new account"""
-    email = click.prompt("Enter your email")
-    company = click.prompt("Enter your associated company")
-    name = click.prompt("(Optional) Enter your name", default="", show_default=False)
-    slug = click.prompt(
-        "(Optional) Enter an unique human-readable identifier for your account",
-        default="",
-        show_default=False,
-    )
-    with Spinner(description="Creating new account"):
-        data = controller.new_account(
-            user_email=email, company=company, user_name=name, slug=slug
-        )
-    return data, "Check your email to verify your account"
+def describe_account(controller):
+    """Get account details"""
+    with Spinner(description="Fetching account details"):
+        return controller.get_account()
+
+
+@iam.command("list-accounts")
+@click.pass_obj
+@pretty_print
+def describe_account(controller):
+    """List all accounts for your user"""
+    with Spinner(description="Fetching all accounts for your user"):
+        return controller.list_accounts()
+
+
+@iam.command("admin-reset-password")
+@click.argument("email")
+@click.pass_obj
+@pretty_print
+def admin_reset_password(controller, email):
+    """Reset a user's password as admin"""
+    with Spinner(description="Resetting user's password as admin"):
+        return controller.admin_reset_password(email=email)
```

Comment:
return an extra message too `Have {user} check their email to reset their password`